File: JavaScript/Faccat/Exercicio16.js
let numeroDeMacas;
let preco;

numeroDeMacas = parseInt(prompt("Digite o número de maçãs compradas: "));

if (numeroDeMacas < 12) {
    preco = numeroDeMacas * (13 / 10);
    console.log("O custo total da compra foi de: R$ " + preco.toFixed(2));
} else {
    preco = numeroDeMacas;
    console.log("O custo total da compra foi de: R$ " + preco.toFixed(2));
}

File: Java/src/Faccat/Exercicio16.java
package Faccat;

import java.util.Scanner;

public class Exercicio16 {
    public static void main(String[] args) {
        int numeroDeMacas;
        double preco;

        Scanner scanner = new Scanner(System.in);
        System.out.println("Digite o número de maçãs compradas: ");
        numeroDeMacas = scanner.nextInt();

        if (numeroDeMacas < 12) {
            preco = numeroDeMacas * (13.0 / 10.0);
            System.out.println("O custo total da compra foi de: R$" + String.format("%.2f", preco));
        } else {
            preco = numeroDeMacas;
            System.out.println("O custo total da compra foi de: R$" + String.format("%.2f", preco));
        }
    }
}
